app/api/crud.py

```diff
@@ -89,7 +96,7 @@ async def get_terms(data_element_URI: str):
     cross_node_results = []
     params = {data_element_URI: data_element_URI}
 
-    for node_url in util.parse_nodes_as_dict(util.NEUROBAGEL_NODES).keys():
+    for node_url in util.FEDERATION_NODES:
```

Comment:
does this mean that the `get_terms` request will always go out to **all** known nodes, regardless of any subselection the user might have made? in other words, wouldn't we also want to restrict this by 

```python
node_urls : list[str]
        List of Neurobagel nodes to send the query to.
```
as above in `async def get`?

Reply:
I've opened a separate issue for this
- https://github.com/neurobagel/federation-api/issues/36

as I think this could benefit from some further discussion.

Right now, I think only the query tool uses this endpoint (`/attributes/{data_element_URI}`) to get all the instances of a given controlled term like `nb:Assessment` (across the known graphs) to populate the dropdown options for the UI. I imagine this would be separate from/before the user actually selects specific nodes in the app to send a query to - not sure if we want the query tool to have dynamically changing dropdown options as a result.